### ISSUE TITLE
Commit messages limited to the last commit

### ISSUE THREAD

**Issue by deimosfr** [goreleaser/goreleaser-action#56]
Hi,

I'd like to have the full changelog in the Github release between versions. When I run goreleaser with action, there is only the last one, however when I'm running it locally, it works as it should. Are this limitations due to Action?

Do you have any suggestions?

Thanks

**Comment by crazy-max** [comment]
Hi @deimosfr,

Can you post your workflow here please and/or link to your repo?

Thanks

**Comment by deimosfr** [comment]
Sure,

Here is the workflow:
```yaml
name: goreleaser

on:
  create:
    tags:

jobs:
  qovery:
    runs-on: ubuntu-latest
    steps:
      -
        name: Checkout
        uses: actions/checkout@master
      -
        name: Set up Go
        uses: actions/setup-go@master
        with:
          go-version: 1.13.x
      -
        name: Run GoReleaser
        uses: goreleaser/goreleaser-action@v1
        with:
          version: latest
          args: release --rm-dist
        env:
          GITHUB_TOKEN: ${{ secrets.GORELEASER_GITHUB_TOKEN }}
```

And link to the release page (and repo): https://github.com/Qovery/qovery-cli/releases

**Comment by crazy-max** [comment]
@deimosfr The issue is [here](https://github.com/Qovery/qovery-cli/commit/6019d6008095531b57054b1a856b71df58bc8e5e/checks?check_suite_id=372685639#step:2:21):

`/usr/bin/git -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=1 origin +6019d6008095531b57054b1a856b71df58bc8e5e:refs/tags/v0.1`

You are using [`actions/checkout@master`](https://github.com/actions/checkout/) which is based on [version 2](https://github.com/actions/checkout/releases/tag/v2.0.0) and the _default behavior now fetches only the commit being checked-out_ as explained in their changelog.

To summarize, you should only use tag names or commit ID when using actions, this will prevent you from having regressions in your workflow. I will update our example in the README.

To fix this on your side, two possibilities:

```
-
  name: Checkout
  uses: actions/checkout@v1
```

or

```
-
  name: Checkout
  uses: actions/checkout@v2
  with:
    fetch-depth: 0
```
